docs/zk-passport/guide-proof-of-citizenship.mdx

```diff
@@ -40,65 +42,68 @@ Links for installation:
-      event_id: "citizenship_verification",
-      // Let a single passport pass verification just once
-      uniqueness: true
-    }
-  }
-}
+If you don't use React, you can use the `@rarimo/zk-passport` package instead. It provides a low-level API for for requesting and passing for verification of ZK Passport proofs.
 
-const response = await fetch(
-
-  `${verificatorUrl}/integrations/verificator-svc/private/verification-link`,
-  {
-    headers: {
-      'Accept': 'application/vnd.api+json',
-      'Content-Type': 'application/vnd.api+json',
-    },
-    method: "POST",
-    body: JSON.stringify(body),
-  }
-)
-const { data } = await response.json()
-// Proof Params URL for RariMe app
-const proofParamsUrl = data.attributes.get_proof_params
-const encodedUrl = encodeURIComponent(proofParamsUrl)
-// Render this data in a QR code:
-const qrCodeContent = `rarime://external?type=proof-request&proof_params_url=${encodedUrl}`)
+```bash npm2yarn
+npm install @rarimo/zk-passport
 ```
 
-Some notes:
+## Step #3: Import the library and render the QR code
+
+The following JS snippet uses the library to render a QR code with a proof of citizenship request and handles the results:
 
-- `event_id` is an arbitrary string that denotes the application scope of the proof, should be something like `APP_ID:ACTION_ID`
-- `id` is a public unique ID associated with a user within your system. It will be used in Step #3 to check the user status
+```jsx
+import ZkPassportQrCode, { ProofRequestStatuses } from '@rarimo/zk-passport-react'
 
-After scanning the QR code, the user will complete the verification in RariMe App.
+// this is a public instance of verifier-svc. Replace it with the URL of your instance
+const apiUrl = 'https://api.app.rarime.com'
 
-## Step #3: Check the user status
+// `requestid` is a public unique ID associated with a user within your system. It will be used in the Step #3 to check the user status
+const requestId = 'account-1'
 
-You can fetch the verification status for a particular `id` from this endpoint:
+// `event_id` is an arbitrary string that denotes the application scope of the proof, should be something like `APP_ID:ACTION_ID`
+const eventId = 'my_dapp'
+
+const verificationOpts: RequestVerificationLinkOpts = {
+  uniqueness: true, // don't allow to submit a single passport multiple times
+  nationalityCheck: true, // include user's citizenship in the output
+  eventId: eventId, // scope of the proof
+}
+
+return (
+  <ZkPassportQrCode
+    // Proof request params
+    apiUrl={apiUrl}
+    requestId={requestId}
+    verificationOptions={verificationOpts}
+
+    // QR code component props
+    qrProps={{ size: 256 }}
+
+    // place your callbacks here
+    onStatusChange={status => console.log(status)}
+    onSuccess={proof => console.log(proof)}
+    onError={error => console.error(error)}
+  />
+)
+```
+
+## Step #3: Fetch the verification status and the citizenship code on the backend
```

Comment:
step 3 is used twice

also I would split these approaches because they are different:
1. Raw verificator-svc requests with fetch (old docs)
2. Using `@rarimo/zk-passport` to encapsulate the verificator-svc logic
3. Using `@rarimo/zk-passport-react` as a simple way to verify proofs within a React app